dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/Result.java

```diff
@@ -36,13 +47,17 @@
      */
     Object getValue();
 
+    void setValue(Object value);
```

Comment:
Why do you not add the default method?

Reply:
The reason is that I think Result is not an API or SPI open to end user. In this way, we can make sure every implementation can aware of the newly added method.